modules/ROOT/nav.adoc

```diff
@@ -80,24 +75,26 @@
  ** xref:add-another-output-transform-studio-task.adoc[Add Another Output Target]
  ** xref:move-transformations-separate-file-studio-task.adoc[Move Transformations to Separate Files]
 
-* xref:eu-cloud-configuration.adoc[Configure EU Cloud in Studio]
-
-* xref:pce-configuration.adoc[Configure Private Cloud Edition in Studio]
- ** xref:pce4-configuration.adoc[Configure PCE 4 in Studio]
- ** xref:pce3-configuration.adoc[Configure PCE 3 in Studio]
- *** xref:pce-configuration-macos.adoc[MacOS]
- *** xref:pce-configuration-windows.adoc[Windows]
-
 * xref:login-platform-from-studio.adoc[]
  ** xref:set-credentials-in-studio-to.adoc[]
 
+* xref:set-anypoint-platform-cloud.adoc[Set Anypoint Platform Cloud]
+ ** xref:deploy-mule-application-task.adoc[Deploy a Mule Application to CloudHub]
+ ** xref:domain-support-concept.adoc[Domain Support]
```

Comment:
Could you add more context here? I'm not sure what the topic is about based on the title. What is the user value?

Reply:
Yes. Domain Support doesn't tell me much about what's included in the topic. The user task seems to be "share global configurations between different Mule applications implementing the same domain"

Reply:
Yes! The user task you described is accurate but maybe too long.
What about?:
"Manage Mule Domains Through Global Elements and Configuration XML".
I can't think of a short and sweet title :/ 
Thanks!